src/runner/provider/qti.js

```diff
@@ -113,10 +113,16 @@ var qtiItemRuntimeProvider = {
                     }
                 }
 
+                const itemPostRender = this._item.postRender(options);
+                let postRenderPromises = [];
+                if (itemPostRender.length) {
+                    postRenderPromises = itemPostRender.filter(render => render instanceof Promise);
```

Comment:
question: since`Promise.all` supports non-Promise items, why do you filter them out?

Reply:
When the promise tries to call and you send other elements, it calls the error, if you remove the other elements and just pass the promise it does not throw the error.